### PR TITLE
iframe comm

### PR DESCRIPTION
**Pull request by billythedummy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,14 +7,15 @@
...
         <script src="minimalTest.js"></script>
+        <script src="gameComm.js"></script>
         <link href="styles/style.css" rel="stylesheet" />
     </head>
     <body>
         <div id="wrapper">
             <h1 id="title"> Omnispace </h1>
             <div id="game">
                 <iframe id="game_frame" allow="cross-origin-isolated"
-                src = "https://telegram-space-l1c9paxke-igneous-labs.vercel.app/">
```
Comment (fplord) [inline index.html:17]: can we revert this change? I don't actually have a local version running on my surface

File: minimalTest.js [hunk 2]
```diff
@@ -92,18 +104,24 @@ async function start() {
         setRoomList();
     })
 
-    client.on("Room.timeline", (event, room, toStartOfTimeline) => {
+    client.on("Room.timeline", (event, room, toStartOfTimeline, _removed, data) => {
```
Comment (fplord) [inline minimalTest.js:107]: what do these params do?

Comment (billythedummy) [reply minimalTest.js:107]: i thought [data.liveEvent](https://matrix-org.github.io/matrix-js-sdk/23.3.0/interfaces/IRoomTimelineData.html#liveEvent) could be used to determine whether an event was live or not instead of using the timestamp trick, but it seems to be true for every single event

Comment (billythedummy) [reply minimalTest.js]: reverted
